### PR TITLE
Added Gloo with AWS ELB documentation

### PR DESCRIPTION
**Pull request by scranton**

### CODE REVIEW

File: docs/content/gloo_integrations/aws/_index.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+Gloo's managed Envoy proxies are installed on EKS as a `LoadBalancer` type Service named `gateway-proxy`. Gloo's Helm chart allows the user to specify annotations to be added to Gloo's `gateway-proxy` Service, including adding AWS ELB annotations that influence the AWS ELB associated with the Gloo proxy service.
+
+The most commonly used AWS annotations used with Gloo are:
+
+* `service.beta.kubernetes.io/aws-load-balancer-type` - the only acceptable value is `nlb` to associate an AWS Network Load Balancer with the Service. If this annotation is not present, then AWS will associate a Classic ELB with this Service.
+* `service.beta.kubernetes.io/aws-load-balancer-internal` (default: `false`) - controls whether AWS ELB will be Public/Internet facing (`true`) or VPC internal (`false`).
+* `service.beta.kubernetes.io/aws-load-balancer-ssl-cert` - If specified, AWS ELB is configured listener uses TLS/HTTPS with the provided certificate. Value is a valid certificate ARN from AWS Certificate Manager or AWS IAM, e.g. `arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012`.
```
Comment (christian-posta) [inline docs/content/gloo_integrations/aws/_index.md]: Check this sentence "AWS ELB is configured listener uses..."

File: docs/content/gloo_integrations/aws/_index.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    gatewayProxy:
+      service:
+        extraAnnotations:
+          service.beta.kubernetes.io/aws-load-balancer-type: nlb
+EOF
+{{< /tab >}}
+
+{{< /tabs >}}
+
+## Pass thru TLS
```
Comment (christian-posta) [inline docs/content/gloo_integrations/aws/_index.md]: Passthrough

File: docs/content/gloo_integrations/aws/_index.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+By using an AWS Network Load Balancer (NLB) in front of Gloo, you get an additional benefit of TLS pass thru. That is, HTTPS requests pass thru the AWS NLB and terminate TLS at the Gloo proxy for extra security. AWS NLB automatically configures listeners for each Kubernetes Service port, so both HTTP and HTTPS ports are exposed through the AWS NLB.
```
Comment (christian-posta) [inline docs/content/gloo_integrations/aws/_index.md]: passthrough

**Comment by scranton** [comment]
fixes https://github.com/solo-io/customers/issues/4

**Pull request merged**